n2y/converter.py

```diff
@@ -31,61 +31,32 @@
 
 
 def load_plugins(filename):
-    global Bookmark, BulletedList, BulletedListItem, ChildPageBlock, CodeBlockFenced, \
-        Divider, HeadingOne, HeadingTwo, HeadingThree, ImageBlock, NumberedList, NumberedListItem, \
-        ParagraphBlock, Quote, RowBlock, TableBlock, ToDo, ToDoItem, Toggle
     abs_path = path.abspath(filename)
     plugin_spec = importlib.util.spec_from_file_location("plugins", abs_path)
     plugin_module = importlib.util.module_from_spec(plugin_spec)
     plugin_spec.loader.exec_module(plugin_module)
     for (key, value) in plugin_module.exports.items():
-        if key == "Bookmark":
-            Bookmark = value
-        elif key == "BulletedList":
-            BulletedList = value
-        elif key == "BulletedListItem":
-            BulletedListItem = value
-        elif key == "ChildPageBlock":
-            ChildPageBlock = value
-        elif key == "CodeBlockFenced":
-            CodeBlockFenced = value
-        elif key == "Divider":
-            Divider = value
-        elif key == "HeadingOne":
-            HeadingOne = value
-        elif key == "HeadingTwo":
-            HeadingTwo = value
-        elif key == "HeadingThree":
-            HeadingThree = value
-        elif key == "ImageBlock":
-            ImageBlock = value
-        elif key == "NumberedList":
-            NumberedList = value
-        elif key == "NumberedListItem":
-            NumberedListItem = value
-        elif key == "ParagraphBlock":
-            ParagraphBlock = value
-        elif key == "Quote":
-            Quote = value
-        elif key == "RowBlock":
-            RowBlock = value
-        elif key == "TableBlock":
-            TableBlock = value
-        elif key == "ToDo":
-            ToDo = value
-        elif key == "ToDoItem":
-            ToDoItem = value
-        elif key == "Toggle":
-            Toggle = value
+        if key in globals():
+            # plugins can only override classes in this file that are derrived from a Block
+            if globals()[key].__name__ in [b.__name__ for b in value.__bases__] \
```

Comment:
I think this line can be simplified by using some intermediate variables (e.g., for `globals()[key]` and `[b.__name__ for b in value.__bases__]`.